docs/topic-guides/blendable-web-apps/common-look-and-feel.md

```diff
@@ -23,43 +23,43 @@ As an app author, you provide the views for your apps with a clear separation of
 
-As an app author, you are invited to implement specialized components in your app that use Uniform internally. This will allow to adapt to the solution’s look and feel also for your specialized components.
+As an app author, you can implement specialized components in your app that use Uniform internally. This lets you adapt specialized components to the solution’s look and feel.
 
 ### Theme configuration through custom CSS properties
 
-Uniform design system comes with a modern-looking default theme.
+Uniform comes with a modern-looking default theme.
 
-As a solution owner, you can customize it by providing new values for the [custom CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables). This allows changing the fonts, colors, etc. without replacing the stylesheet.
+As a solution owner, you can customize the theme primitives by providing new values using [custom CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables). This allows changing the fonts, colors, and other values without replacing the stylesheet.
 
 You can provide new values directly in a stylesheet in a custom composition. It requires some skill in CSS and HTML but allows for spectacular results such as providing a theme configuration only for a specific part of the app.
 
-If you don't need that much of control and prefer a simple to use GUI, use the "Theme configurations" feature of [Blending](https://github.com/Starcounter/Blending) app suite. It is an administration tool that stores the values of custom CSS properties in the database. It can provide theme configurations for specific layouts that affect all Uniform components on the screen.
+If you don't need that much of control and prefer a simple to use GUI, use the "Theme configurations" feature of the [Blending](https://github.com/Starcounter/Blending) app suite. It is an administration tool that stores the values of custom CSS properties in the database. It can provide theme configurations for specific layouts that affect all the Uniform components on the screen.
 
 {% hint style="warning" %} Attaching custom CSS properties through "Theme configurations" sets them on the global scope - the `body` of the light DOM (https://github.com/Starcounter/Blending/issues/232). Providing custom CSS properties values in custom compositions attaches them in shadow DOM to the scope that you choose. {% endhint %}
 
-{% hint style="info" %} For more sophisticated theming we would love to use native [CSS shadow parts and themes](https://meowni.ca/posts/part-theme-explainer/). Unfortunately, it's not supported yet by any browser. {% endhint %}
+{% hint style="info" %} For more sophisticated themeing we would love to use native [CSS shadow parts and themes](https://meowni.ca/posts/part-theme-explainer/). Unfortunately, it's not supported yet by any browser. {% endhint %}
```

Comment:
Why did you change spelling to "themeing"? Others spell it "theming", see, for example [material design](https://material.io/design/material-theming/overview.html#material-theming)

Reply:
Late hour... Thanks for noticing, I will fix